### PR TITLE
Add LONG Token on zkSync Era network (Update ERA.json)

### PR DESCRIPTION
**Pull request by ZkSwapFinance**
Website: https://long.fun/
Coinmarketcap: https://coinmarketcap.com/currencies/long-fun/
Coingecko: https://www.coingecko.com/en/coins/long-fun

### CODE REVIEW

**Review by JasnaLiFi — APPROVED** [review]

File: tokens/ERA.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
        "decimals":18,
        "name":"Tarot",
        "logoURI":"https://static.debank.com/image/op_token/logo_url/0x375488f097176507e39b9653b88fdc52cde736bf/279c43a0188f1a56de7921e454091a52.png"
+    },
+    {
+       "address":"0x5165ec33b491d7b67260B3143f96Bb4aC4736398",
+       "chainId":324,
+       "symbol":"LONG",
+       "decimals":18,
+       "name":"LONG",
```
Comment (myz1237) [inline tokens/ERA.json:15]: According to the debank information, the name is `Long`.

Original return:
```json
{
   "id":"0x5165ec33b491d7b67260b3143f96bb4ac4736398",
   "chain":"era",
   "name":"Long",
   "symbol":"LONG",
   "display_symbol":null,
   "optimized_symbol":"LONG",
   "decimals":18,
   "logo_url":"https://static.debank.com/image/era_token/logo_url/0x5165ec33b491d7b67260b3143f96bb4ac4736398/3e61868b9b13b95a7e6809647050f474.png",
   "protocol_id":"",
   "price":1.4259773351751368e-06,
   "price_24h_change":0.05325374617979149,
   "credit_score":29132.19030253106,
   "is_verified":true,
   "is_scam":false,
   "is_suspicious":false,
   "is_core":true,
   "is_wallet":true,
   "time_at":1711191402.0,
   "low_credit_score":false
}
```

File: tokens/ERA.json [hunk 2]
```diff
@@ -6,5 +6,13 @@
...
+       "logoURI":"https://static.debank.com/image/era_token/logo_url/0x5165ec33b491d7b67260b3143f96bb4ac4736398/3e61868b9b13b95a7e6809647050f474.png"
     }
- ]
\ No newline at end of file
+ ]
```
Comment (myz1237) [inline tokens/ERA.json:18]: please check the format of this bracket

**Review by myz1237 — CHANGES_REQUESTED** [review]
commented

**Comment by ZkSwapFinance** [comment]
> commented

Is everything okay? Do we need to create a new PR to change the 'name' field?

**Comment by myz1237** [comment]
> > commented
> 
> Is everything okay? Do we need to create a new PR to change the 'name' field?

Could you change it in your pr and push commits here? Change the name and bracket here: 
image

Thanks in advance

**Comment by ZkSwapFinance** [comment]
> > > commented
> > 
> > 
> > Is everything okay? Do we need to create a new PR to change the 'name' field?
> 
> Could you change it in your pr and push commits here? Change the name and bracket here: image
> 
> Thanks in advance

I pushed a new commit with the name field changed. Could you verify it again?

**Review by myz1237 — APPROVED** [review]

**Pull request merged**